content/en/docs/Components/Filters/MetadataLastModified.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "MetadataLastModified"
+weight: 10
+date: 2020-02-24
+---
+
+{{% pageinfo color="primary" %}}
+
+**Read the [API documentation &raquo;](https://pkg.go.dev/github.com/AdRoll/baker/filter#MetadataLastModified)**
+{{% /pageinfo %}}
+
+## Filter _MetadataLastModified_
+
+### Overview
+
+Extract the last modified timestamp from the record Metadata and write it on a selected field.
```

Comment:
```suggestion
Extract the "last modified" timestamp from the record Metadata and write it to the selected field.
```

Reply:
I'd maybe add some more info or links about this "last modified" timestamp. Reading this doc isn't really clear what we're talking about